googleCalender2.js

```diff
@@ -6,53 +6,53 @@ let max = 1;
 
 async function usersFullSync({ dataStore, client }) {
   for (let i = 1; i <= 3; i++) {
```

Comment:
IS this for loop is required because I didn't find the use of this loop inside the function?